assets/js/chart_munger.js

```diff
@@ -0,0 +1,44 @@
+var munge_data = function(chart_data, chart_name) {
+  // var config = chart_data.config;
```

Comment:
Well I've been assuming all along that we'd want to go back in and add color specifications per-dataset (like to distinguish "Actual" and "Projected" by color on the bar chart), so I left those in. I had it working pre-refactor, but hadn't gotten around to reimplementing it. I can take them out.